auth/authRouter.spec.js

```diff
@@ -4,28 +4,33 @@ const Users = require("../users/users-model");
 const db = require("../data/db-config");
 
 describe("Users Login & Register", () => {
-  // Clears the users table when a test is ran to ensure that the db is clean
-  afterAll(async () => {
-    await db("users").truncate();
+
+  let newUserID
+
+  beforeAll(async () => {
+
   });
 
-  afterAll(done => {
-    return server && server.close(done);
+  afterAll(async () => {
+    await Users.deleteUser(newUserID)
   });
 
   describe("Logs user in", () => {
     beforeAll(async () => {
-      await db("users").truncate();
+      await db("users")
     });
 
     test("Should allow a new user to be created", async () => {
       // Registers a new user;
 
-      const user = { email: "test@test.com", password: "password" };
+      const user = { email: "test@test.com", password: "password", first_name: "greg", last_name: "lala" };
       const response = await request(server)
         .post("/api/auth/register")
         .send(user);
 
+      console.log('userCreate', response.body, response.status)
```

Comment:
I think if some console logs are vital that maybe we could set up some sort of conditional that executes them only when we're in a dev environment.